### PR TITLE
zh-cn: Format /learn/{!css,html,javascript,server-side} using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats various folders and documents throughout the /learn folder, skipping over the big ones for now, for the Simplified Chinese locale.

### CODE REVIEW

**Comment by yin1999** [comment]
I would update the document (`files/zh-cn/learn/forms/how_to_build_custom_form_controls/index.md`) to fix the build error. It may take a few days.

**Comment by queengooborg** [comment]
Thanks for tackling that, @yin1999!

This same issue with the PR test failing for unrelated reasons came up in another PR for another locale -- we decided that it would be better to merge the Prettier PR as-is and worry about the flaw at a later date.  Should we do the same here?

File: files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md [hunk 1]
```diff
@@ -58,25 +58,25 @@ web 基于非常基本的客户端/服务器体系结构，可以总结如下:
 在这个例子中，数据被发送到一个绝对 URL —— `http://foo.com`：
 
 ```html
-<form action="http://foo.com">
+<form action="http://foo.com"></form>
```
Comment (yin1999) [inline files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md:61]: coud we also add `…`

image

```suggestion
<form action="http://foo.com">…</form>
```

File: files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md [hunk 2]
```diff
@@ -58,25 +58,25 @@ web 基于非常基本的客户端/服务器体系结构，可以总结如下:
...
 ```
 
 这里，我们使用相对 URL——数据被发送到服务器上的不同 URL
 
 ```html
-<form action="/somewhere_else">
+<form action="/somewhere_else"></form>
```
Comment (yin1999) [inline files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md:67]:
```suggestion
<form action="/somewhere_else">…</form>
```

File: files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md [hunk 3]
```diff
@@ -58,25 +58,25 @@ web 基于非常基本的客户端/服务器体系结构，可以总结如下:
...
 ```
 
 在没有属性的情况下，像下面一样，{{HTMLElement("form")}}数据被发送到表单出现的相同页面上：
 
 ```html
-<form>
+<form></form>
```
Comment (yin1999) [inline files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md:73]:
```suggestion
<form>…</form>
```

File: files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md [hunk 4]
```diff
@@ -58,25 +58,25 @@ web 基于非常基本的客户端/服务器体系结构，可以总结如下:
...
 ```
 
 许多较老的页面使用下面的符号表示数据应该被发送到包含表单的相同页面；这是必需的，因为直到 HTML5[`action`](/zh-CN/docs/Web/HTML/Element/form#action)属性都需要该符号。现在，这不再需要了。
 
 ```html
-<form action="#">
+<form action="#"></form>
```
Comment (yin1999) [inline files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md:79]:
```suggestion
<form action="#">…</form>
```

**Review by yin1999 — APPROVED** [review]
Thanks. Just some suggestions. We could land this after that :)

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**